loader/src/main/java/net/minecraftforge/fml/loading/FMLPaths.java

```diff
@@ -63,10 +65,12 @@ public static void loadAbsolutePaths(Path rootPath)
             {
                 try {
                    Files.createDirectories(path.absolutePath);
-                } catch (IOException e) {
-                   throw new RuntimeException(e);
+                } catch (IOException ioe) {
+                    throw new UncheckedIOException(ioe);
                 }
             }
+            if (path.isDirectory && Files.exists(path.absolutePath) && !Files.isDirectory(path.absolutePath)) {
```

Comment:
Is this `if` statement meant to be wrapping something? because right now it's just an empty body.